src/engine/Operation.h

```diff
@@ -284,7 +282,8 @@ class Operation {
   template <typename F>
   void forAllDescendants(F f) const;
 
-  RuntimeInformation _runtimeInfo;
+  std::shared_ptr<RuntimeInformation> _runtimeInfo = nullptr;
+  std::shared_ptr<RuntimeInformation> _rootRuntimeInfo = nullptr;
```

Comment:
I'll have a look at why they should default to `nullptr`, and cannot be immediately set.